lua/calcium/scratchpad.lua

```diff
@@ -72,13 +72,16 @@ function M.evaluate_scratchpad()
 
 	vim.api.nvim_buf_clear_namespace(state.buf, state.ns, 0, -1)
 
+	local previous_evaluation = nil
 	local variables = calculator.extract_variables(lines)
 
 	for i, line in ipairs(lines) do
 		if line ~= "" then
+			variables["_"] = previous_evaluation
```

Comment:
Rename to `ans` to follow `qalc` (cli-tool) standard. If you believe `_` is better, implement both.
```suggestion
			variables["ans"] = previous_evaluation
```

Reply:
I think `ans` is more standard here :+1: 
But seems very simple to make this configurable too